assets/js/components/Collection/Collection.test.jsx

```diff
@@ -1,10 +1,49 @@
 import React from "react";
```

Comment:
@katdivyareddy10 Can we add some more tests here which validate that props are passing through and displaying (ie. Description, Admin Email) in the component?

And also maybe test that the Collection image (or placeholder at least) is present.   And that the component displays the `CollectionSearch` component?